### PR TITLE
[docs] fix typo in expo-video page

### PR DESCRIPTION
**Pull request by bishaldahal**
# Why

Found the typo while reading the documentation.

# How

Fixed the typo in SharedObject.ts of expo/packages/expo-modules-core/build/ts-declarations/SharedObject.ts

# Test Plan

# Checklist

- [x] Documentation is up to date to reflect these changes (eg: https://docs.expo.dev and README.md).
- [x] Conforms with the [Documentation Writing Style Guide](https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md)
- [x] This diff will work correctly for `npx expo prebuild` & EAS Build (eg: updated a module plugin).

### CODE REVIEW

**Review by amandeepmittal — CHANGES_REQUESTED** [review]
Hey @bishaldahal, appreciate that you are fixing the issue in the doc. However, it's unclear to us what has been fixed in this PR. Directly changing the generated file is something we often avoid for published SDK docs. 

A good a way to update/fix something in this doc (applies to most published SDK reference docs), is to checkout the the sdk-51 branch, make the changes you to the package as per the instructions describe here: https://github.com/expo/expo/tree/main/docs#update-api-reference-docs, stash the docs source file generated and create the PR branch with the those changes.

It'll be also nice to so share (maybe a screenshot or describe the issue as per the template) as to what issue is being fixed in the process.

**Comment by alanjhughes** [comment]
@bishaldahal can you change the base branch to `main`

**Comment by bishaldahal** [comment]
> @bishaldahal can you change the base branch to `main`

OK will do that.

**Comment by Simek** [comment]
Can you revert the docs data changes for SDK 51? If those changes are needed, file should be regenerated from SDK branch)? 

The `unversioned` file should be updated instead, you can check the API documentation guide we have in Readme to help you with the process:
* https://github.com/expo/expo/tree/main/docs#update-api-reference-docs

**Comment by Simek** [comment]
To fix the SDK CI you also need to rebuild the package locally, and commit the `build` files. You can read more about this, and how to contribute to the repository in our guide:
* https://github.com/expo/expo/blob/main/CONTRIBUTING.md#%EF%B8%8F-editing-sdk-packages

**Comment by bishaldahal** [comment]
> To fix the SDK CI you also need to rebuild the package locally, and commit the `build` files. You can read more about this, and how to contribute to the repository in our guide:
> 
> * https://github.com/expo/expo/blob/main/CONTRIBUTING.md#%EF%B8%8F-editing-sdk-packages

Can you recheck the requirements?

**Review by Simek — APPROVED** [review]
Thanks for all of the updates! 👍 

Let's revert SDK 51 file again before we ship, since there were already changes to the package which will not be cherry-picked to the current stable release.

**Comment by bishaldahal** [comment]
@Simek I am still confused. Can you provide instructions if anything is to be done?

**Comment by Simek** [comment]
No worries, let me rephrase and sum up!

I have asked you before to revert the SDK 51 data file, which you have done:
* https://github.com/expo/expo/pull/29755#issuecomment-2169278949

However, after further changes and rebases the file is present again in the change set:
* https://github.com/expo/expo/pull/29755/files#diff-3e3bf9e67dbe656f8c71c67b65a72bc42708375c679e950fa9f4bc31a56f31c9

Now, I'm asking you to revert it once more, so we can ship your changes. I have stated the reasoning why we should not update that file in the previous comment.

**Comment by Simek** [comment]
There are still changes in the SDK 51 data file, please revert all changes to that file, so no diff is produced.

Screenshot 2024-06-30 202025

**Comment by bishaldahal** [comment]
@Simek Can you please review this?

**Comment by Simek** [comment]
There is a conflict in a branch, we cannot merge the changes, until it's resolved.

Screenshot 2024-07-13 213423

**Comment by Simek** [comment]
Thanks for all of the updates and patience with followups @bishaldahal, great work! 👍

**Review by Simek — APPROVED** [review]
